dotCMS/src/main/webapp/WEB-INF/messages/Language.properties

```diff
@@ -523,7 +523,7 @@ com.dotcms.repackage.javax.portlet.title.link-checker=Link Checker
 com.dotcms.repackage.javax.portlet.title.NetworkPortlet=Network
 com.dotcms.repackage.javax.portlet.title.es-search=ES Search
 com.dotcms.repackage.javax.portlet.title.EXT_LICENSE_MANAGER=Licensing
-
+com.dotcms.repackage.javax.portlet.title.content-types-angular = Content Types Angular
```

Comment:
This is a 4.2 only change and should live only in master, I think something went wrong with a merge